addons/l10n_nl_edi/tests/test_nl_edi_invoicing.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+from odoo import Command
+from odoo.tests import tagged
+from odoo.addons.account_edi.tests.common import AccountEdiTestCommon
+from lxml import etree
+
+
+@tagged('post_install_l10n', '-at_install', 'post_install')
+class TestNlEdiInvoicing(AccountEdiTestCommon):
+
+    @classmethod
+    def setUpClass(cls, chart_template_ref=None, edi_format_ref=None):
+        super().setUpClass()
+
+        cls.fixed_tax = cls.env['account.tax'].create({
+            'name': 'Test Tax',
+            'amount_type': 'fixed',
+            'description': 'test label',
+            'include_base_amount': True,
+            'country_id': cls.env.ref('base.nl').id,
+            'amount': 10.0,
+        })
+
+        cls.company_data['company'].write({
+            'name': 'NL Company Test',
+            'phone': '+31 6 12345678',
+            'vat': 'NL000043603B11',
+            'zip': '9700',
+            'email': 'info@company.nlexample.com',
+            'country_id': cls.env.ref('base.nl').id,
+        })
+
+        cls.journal.write({
+            'edi_format_ids': [
+                cls.env['account.edi.format'].search([('code', '=', 'ubl_bis3')], limit=1).id,
+                cls.env['account.edi.format'].search([('code', '=', 'nlcius_1')], limit=1).id,
+            ]
+        })
+
+    def test_fixed_tax_nlcius_and_peppol(self):
+        """
+        Ensure that an invoice containing a product with a fixed tax posted to a journal with the peppol and nlcius edi
+            tags generates edi documents with accurate LineExtensionAmount values
+        """
+        invoice = self.env['account.move'].create({
+            'move_type': 'out_invoice',
+            'invoice_date': '2000-05-05',
+            'partner_id': self.partner_a.id,
+            'invoice_line_ids': [Command.create({
+                'name': 'product costing 50.0',
+                'quantity': 1,
+                'price_unit': 50.0,
+                'tax_ids': [Command.set([self.fixed_tax.id])],
```

Comment:
I don't think this is a valid business case. You would need to add an additional percent tax on the line. Otherwise, I think you will not be able to generate the xml in 16.2 (Bis 3 expects one percent tax per line, and raise a UserError otherwise).